packages/paste-website/src/components/icons/PasteIconInverse.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import {useUID} from 'react-uid';
+
+export interface PasteIconInverseProps {
+  className?: string;
+  color?: string;
+  decorative?: boolean;
+  display?: string;
+  size?: number;
+  title?: string;
+}
+
+const PasteIconInverse = React.memo(
+  ({title = 'Twilio Paste', className, color, decorative = true, display, size}: PasteIconInverseProps) => {
+    const titleId = useUID();
```

Comment:
Is there a way to give id a specific prefix, the id is not unique to the page due to a bug with Paste icons. 

We have 2 sets of useUID starting from 1.

Reply:
`titleId = thing-${useUID()}` I believe